package.json

```diff
@@ -71,7 +72,7 @@
     "test": "jest --coverage",
     "bundle:extension": "node ./esbuild.config.js",
     "vscode:package": "npm run compile && npm run bundle:extension & npx vsce package",
-    "vscode:prepublish": "npm run compile && npm run bundle:extension && npm run update-for-packaging",
+    "vscode:prepublish": "npm run update-for-packaging && npm prune --production",
```

Comment:
So your github action to do the build does an npm ci, compile, then bundle of the code.  Since esbuild is happily bundling everything I'm not sure we even need anything in the dependencies for the packageUpdates section in package.json.  I added the `npm prune --production` before excluding node_module, but I don't think it's needed with vscode ignoring all of node_modules.